e2e/connect.test.tsx

```diff
@@ -12,42 +12,55 @@ import { SwapWidget } from '../src'
 
```

Comment:
This PR should be marked as a draft until the tests are resolved - it shouldn't be merged with tests commented out.